.github/workflows/nightly-docker-build.yaml

```diff
@@ -30,11 +30,19 @@ jobs:
 
       - run: npm ci
 
-      - name: Compile
+      - name: Build
         run: |
-          touch .env
```

Comment:
Just to confirm. Do we no longer need this file? :)

Reply:
We need it. Do not remove it

Reply:
@karl-cardenas-coding  @caroldelwing  Our actions contain a mixture of `make build` and `npm run build` right now. I have changed all actions to use `npm run build` for consistency. This is why the `.env` file is no longer require across our workflows.